src/SmartComponents/Systems/Systems.js

```diff
@@ -218,6 +219,7 @@ const Systems = () => {
                                     selectedTags
                                 },
                                 shouldRefresh: patchSetState.shouldRefresh === true
+                                    || unassignSystemsModalState.shouldRefresh === true
```

Comment:
Wouldn't just `shouldRefresh: patchSetState.shouldRefresh || unassignSystemsModalState.shouldRefresh` be enough?

Reply:
I have used onRefreshData function to relaod Inventory component. This helped to make the code to be readable, I think.